<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fm_Gestoque</class>
 <widget class="QDialog" name="Fm_Gestoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_newproduct">
        <attribute name="title">
         <string>Novo Produto</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Código do Produto:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_cod"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Descrição Produto:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_desc"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Qtde Estoque:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Txt_estoque"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Valor de Compra:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Txt_compra"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Valor de Venda:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Txt_venda"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Fornecedor:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Txt_fornecedor"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="Bt_add">
             <property name="text">
              <string>Novo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Bt_save">
             <property name="text">
              <string>Gravar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_Gproducts">
        <attribute name="title">
         <string>Gestão Produtos</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_19">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Código do Produto:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Txt_Gcod">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Descrição Produto:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Txt_Gdesc"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Qtde Estoque:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_Gestoque"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Valor de Compra:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_Gcompra"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_17">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Valor de Venda:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_Gvenda"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_18">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Fornecedor:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_Gfornecedor"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="Bt_Ggravar">
               <property name="text">
                <string>Gravar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Bt_Gexcluir">
               <property name="text">
                <string>Excluir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_21">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Filtrar por:</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QRadioButton" name="Rb_Gcod">
                  <property name="text">
                   <string>Código Produto</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="Rb_Gnome">
                  <property name="text">
                   <string>Descrição Produto</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_20">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Filtrar:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Bt_Gfiltrar">
                   <property name="styleSheet">
                    <string notr="true">font-size:12pt;</string>
                   </property>
                   <property name="text">
                    <string>Filtrar</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLineEdit" name="Txt_Gfiltrar"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="TableView"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
